test/e2e/allocation.cancel.test.js

```diff
@@ -0,0 +1,22 @@
+import createAllocation from './_allocation'
+import { pmuUser } from './_roles'
+import { allocations, newAllocation } from './_routes'
+import { allocationCancelPage, page } from './pages/'
+
+fixture('Cancel allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+  await createAllocation()
+  await t.useRole(pmuUser).navigateTo(allocations)
```

Comment:
You shouldn't need to use `useRole` again here as you're already signed in using that role.